### PR TITLE
Update README.md

### LINKED ISSUE #37: Turn on subtitles for YouTube videos
If it's difficult to understand my English, please turn on the subtitles in the YouTube videos.  Auto-transcribed subtitles will be shown.

### PR DESCRIPTION
**Pull request by cmpark0126**
- Update `Online Session` notice 
  - Resolve #37 
- Add remote server troubleshooting method related to #91
- Resolve https://github.com/kaist-cp/kecc/issues/272

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,13 +12,14 @@
 
 ### Online sessions
 
-Due to COVID-19, we're going to conduct online sessions at least from 16th to 27th of March.
+Due to COVID-19, we're going to conduct online sessions throughout the semester.
```
Comment (jeehoonkang) [inline README.md:15]: We dont know yet

Comment (cmpark0126) [reply README.md:15]: How about "Due to COVID-19, we're going to conduct online sessions until the next notice of the school."?

Comment (jeehoonkang) [reply README.md:15]: Id rather delete phrases from "at least ...".

Comment (cmpark0126) [reply README.md]: I've fixed it.

**Pull request merged**